### DISCUSSION TITLE
Can I use a port of noisium for forge ?

### DISCUSSION THREAD

**Discussion by coco875** [Steveplays28/noisium#8] [General]
Hi, I make a port to forge in 1.19.2 ([here](https://github.com/coco875/noisium-forge)), I can use it in modpack and I can publish this version on curseforge and modrinth ?

**Comment by Steveplays28** [comment]
Go for it, as long as you correctly link back to the source and the original version of the mod (this repository).
Thanks for checking out Noisium and porting!

I'll look into proper multiplatform support in the future, I'll let you know when that's in place.

**Reply by coco875** [reply]
ok thanks

**Reply by coco875** [reply]
it's fine like this for the mod description ?
![image](https://github.com/Steveplays28/noisium/assets/59367621/baaab762-9786-419f-9348-913583293bf3)

**Reply by Steveplays28** [reply]
Yes, that works. Thank you.

**Comment by Mqony** [comment]
> Hi, I make a port to forge in 1.19.2 ([here](https://github.com/coco875/noisium-forge)), I can use it in modpack and I can publish this version on curseforge and modrinth ?

Hi, do you happen to have a 1.18.2 version of your port? It would be greatly appreciated, thank you

**Reply by coco875** [reply]
I didn't make a 1.18.2 because more and more mod don't support it and for 1.20+ version you have connector who work with nosium so no port are needed
